examples/decoding/plot_decoding_rsa.py

```diff
@@ -0,0 +1,179 @@
+"""Representational Similarity Analysis is used to perform summary statistics
```

Comment:
you need a headline eg.

Representational Similarity Analysis on spiking data
======================================